### ISSUE TITLE
Error TS2349 using concurrently in full ESM module

### ISSUE THREAD

**Issue by throrin19** [open-cli-tools/concurrently#454]
Hello, 

I have a problem using  concurrently (and others modules) in my project after migrate from CJS to full ESM. I have the error TS2349 when I try to call `concurrently` function. But, In VSCode, the definition is correctly loaded on CTRL+click :/

This is my config : 

- tsconfig.json
    ```json
    {
      "compileOnSave": false,
      "compilerOptions": {
        "rootDir": ".",
        "sourceMap": true,
        "declaration": true,
        "esModuleInterop": true,
        "emitDecoratorMetadata": true,
        "experimentalDecorators": true,
        "importHelpers": false,
        "resolveJsonModule": true,
        "noImplicitAny": false,
        "strict": true,
        "useDefineForClassFields": false,
        "target": "ES2022",
        "lib": [
          "esnext"
        ],
        "skipLibCheck": true,
        "skipDefaultLibCheck": true,
        "baseUrl": ".",
        "outDir": "dist",
        "types": [
          "node"
        ],
        "module": "NodeNext",
        "moduleResolution": "NodeNext",
      }
    }
    ```

And my example code : 

```ts
import concurrently from 'concurrently';

const main = async () => {
    concurrently([
        { command : 'ls /etc' },
        { command : 'ls /usr' }
    ]);
}

main();
```

He is a screen on my VSCode : 
![image](https://github.com/open-cli-tools/concurrently/assets/1394778/159f52fa-0782-460f-8ee4-a8c057e8fe07)

I don't know how to do to fix that. I think my problem is with typescript, but I don't understand how to fix that :/

**Comment by gustavohenke** [comment]
Hey! So apparently there's a lengthy discussion here about this issue: https://github.com/microsoft/TypeScript/issues/50058
It's coming from the use of `module: "Nodenext"` in your tsconfig.json.

There are a couple of hacks suggested in there that you could try, but none that is pretty.
It seems that what typescript is doing is right for your compiler options. Concurrently is a commonjs module, [hacked to look like a native ESM module](https://github.com/open-cli-tools/concurrently/blob/v8.0.0/index.mjs).

**Comment by gustavohenke** [comment]
Hey! This is now fixed in v9.0.0.
https://github.com/open-cli-tools/concurrently/releases/tag/v9.0.0
